WordPress/Classes/ViewRelated/Blog/Blog Details/BlogDetailsViewController+FancyAlerts.swift

```diff
@@ -6,6 +6,7 @@ private var observer: NSObjectProtocol?
 extension BlogDetailsViewController {
 
     @objc static let bottomPaddingForQuickStartNotices: CGFloat = 80.0
+    @objc static var shouldScrollToViewSite = false
```

Comment:
I'm not sure about having this as a static var (I presume this is because we can't add instance vars in an extension?). I think I'd prefer to have this as an instance variable defined in BlogDetailsViewController.

Reply:
Agreed!